docs/tempo/website/getting-started/tempo-in-grafana.md

```diff
@@ -55,7 +58,23 @@ Once the requirements are set up, this pre-configured dashboard is immediately a
 
 For more information, refer to the [APM dashboard documentation]({{< relref "../metrics-generator/app-performance-mgmt/" >}}).
 
-<p align="center"><img src="../apm-overview.png" alt="APM dashboard overview"></p>
+<p align="center"><img src="../assets/apm-overview.png" alt="APM dashboard overview"></p>
+
+## Metrics from spans
+
+Requests Error Duration (RED) metrics can be used to drive service graphs and other ready-to-go visualizations of your span data. 
```

Comment:
Should a hyperlink point to some RED content? Assuming there is some standard RED doc :slightly_smiling_face:

Reply:
I've updated this. Thank you for the suggestions.